Editor/Editors/CinemachineFreeLookEditor.cs

```diff
@@ -95,27 +92,33 @@ public override void OnInspectorGUI()
             DrawExtensionsWidgetInInspector();
         }
 
-        string[] RigNames;
-        CinemachineVirtualCameraBase[] m_rigs;
-        UnityEditor.Editor[] m_editors;
-        void UpdateRigEditors()
+        static GUIContent[] s_RigNames = 
         {
-            RigNames = CinemachineFreeLook.RigNames;
-            if (m_rigs == null)
-                m_rigs = new CinemachineVirtualCameraBase[RigNames.Length];
-            if (m_editors == null)
-                m_editors = new UnityEditor.Editor[RigNames.Length];
-            for (int i = 0; i < RigNames.Length; ++i)
+            new GUIContent("Top Rig"), 
+            new GUIContent("Middle Rig"), 
+            new GUIContent("Bottom Rig")
+        };
+        int m_SelectedRig = 1;
```

Comment:
This should be static to be consistent with NewFreelook, and vcams FTransposer or Composer